ground/src/main/java/com/google/android/ground/domain/usecases/submission/SubmitDataUseCase.kt

```diff
@@ -63,13 +63,11 @@ constructor(
       }
     }
 
-    submissionRepository
-      .saveSubmission(
-        surveyId,
-        requireNotNull(loiIdToSubmit) { "No LOI found present for submission" },
-        deltasToSubmit
-      )
-      .blockingAwait()
+    submissionRepository.saveSubmission(
+      surveyId,
+      requireNotNull(loiIdToSubmit) { "No LOI found present for submission" },
```

Comment:
Please add LOI and submission ID for debugging

Reply:
Done